src/components/AddBook.js

```diff
@@ -17,16 +20,14 @@ const AddBook = () => {
     }
   };
 
-  const storeData = () => {};
-
   return (
     <>
       <div className="add-container">
         <h2>ADD NEW BOOK</h2>
         <form action="#" method="post">
           <input type="text" name="title" placeholder="Book Title" onChange={inputData} />
           <input type="text" name="author" placeholder="Book Author" onChange={inputData} />
-          <button type="button" onClick={storeData}>Add Book</button>
+          <button type="button" onClick={()=> dispatch(addBook(infor))}>Add Book</button>
```

Comment:
- [ ] (OPTIONAL) Kindly validate your input fields so that a user would not be able to create a book without a title or author 👍